src/tests/unit/recipeService/getRecipesByOwner.test.ts

```diff
@@ -0,0 +1,52 @@
+import { mockId, saveRecipes } from '../../mocks/mockRecipe';
+import RecipeService from '../../../services/recipeService';
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('getRecipesByOwner', () => {
+  describe('should return', () => {
+    test('existing recipes', async () => {
+      await saveRecipes(3, mockId);
+      const recipes = await recipeService.getRecipesByOwner(mockId);
+      expect(recipes).toHaveLength(3);
```

Comment:
I think it's worth checking that 3 returned recipes are the same as the ones you've added to the database.
Imagine having a bug that does not clear your database completely, and there are some leftovers - maybe the count of recipes will be the same, but the `ownerId` is different?